### PR TITLE
Update Workflows for new Loader

### PR DESCRIPTION
**Pull request by perkinslr**
## Additions

A python script that runs on PRs to check for duplicate xml paths in the virtual filesystem.

## Changes

CombatExtendedLoader.dll gets stripped from built versions.
CompatAssemblies and related resources are removed.

## Reasoning

The loader functionality is now embedded in `CombatExtended.dll`.  The Loader dll still exists to warn people they are running the uncompiled source tree, but it is not needed in a built version, so is removed prior to zipping the mod.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: .github/workflows/duplicates.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Check for duplicate virtual file paths
+      run: |
+        python3 DupeFinder.py -c
```
Comment (MadaraUchiha) [inline .github/workflows/duplicates.yml:20]: Where is `DupeFinder.py` itself?

File: Anomaly/Patches/DamageDefs/Anomaly_Damages_Misc.xml
Comment (MadaraUchiha) [inline Anomaly/Patches/DamageDefs/Anomaly_Damages_Misc.xml:1]: This change seems unrelated to the overarching PR

**Review by MadaraUchiha — APPROVED** [review]
Approved with a couple of questions

File: .github/workflows/duplicates.yml (hunk 1 above)
Comment (perkinslr) [reply .github/workflows/duplicates.yml:20]: It is in the root of the repository, added by 3330.

File: Anomaly/Patches/DamageDefs/Anomaly_Damages_Misc.xml
Comment (perkinslr) [reply Anomaly/Patches/DamageDefs/Anomaly_Damages_Misc.xml:1]: They are required for the new test to not immediately fail, as they are a duplicate identified by DupeFinder
